.github/workflows/cloudgrep-demo.yaml

```diff
@@ -0,0 +1,50 @@
+name: cloudgrep-demo
+
+on:
+  workflow_dispatch:
+    inputs:
+      aws_key_id:
+        description: "AWS key ID"
+        required: true
+      aws_secret_key:
+        description: "AWS secret key"
+        required: true
+      aws_session_token:
+        description: "AWS session token"
+        required: true
+  release:
+    types: [ "published" ]
+
+env:
+  GO_VERSION: 1.18
+  CGO_ENABLED: 0
+
+  AWS_REGION: us-east-1
+
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  deploy-cloudgrep-demo:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ github.event.inputs.aws_key_id }}
+      AWS_SECRET_ACCESS_KEY: ${{ github.event.inputs.aws_secret_key }}
+      AWS_SESSION_TOKEN: ${{ github.event.inputs.aws_session_token }}
+      AWS_REGION: ${{ env.AWS_REGION }}
```

Comment:
We shouldn't use static creds if we can avoid it. We should assume an AWS role from the github action instead. See [this workflow](https://github.com/run-x/cloudgrep/blob/main/.github/workflows/integration-setup.yml#L55-L59) for an example. See [the action](https://github.com/aws-actions/configure-aws-credentials) for more docs.